config/projects/google-fluentd.rb

```diff
@@ -8,13 +8,13 @@
 description "Google Fluentd: A data collector for Google Cloud Logging"
 
 install_dir     "/opt/google-fluentd"
-build_version   "1.5.9"
+build_version   "1.5.9.grpc.alpha.2"
 build_iteration 1
 
 # creates required build directories
 dependency "preparation"
 
-override :ruby, :version => '2.1.10'
+override :ruby, :version => '2.1.8'
```

Comment:
Yeah, we could potentially use 2.1.9. The issue only exists for multi-digit versions. "10" is bad. "9" is fine. LOL I chose "2.1.8" because that was what we upgraded from, and thus should be a relatively stable/tested version.